app.json

```diff
@@ -11,10 +11,13 @@
   },
   "formation": {
     "web": {
-      "quantity": 1
+      "quantity": 1,
+      "size": "free"
     }
   },
   "name": "urlaubsverwaltung",
+  "repository": "https://github.com/synyx/urlaubsverwaltung",
+  "logo": "https://github.com/synyx/urlaubsverwaltung/blob/heroku/src/main/resources/static/favicon.ico",
```

Comment:
Gehört hier in diese Logo-Url ein `heroku`?

Reply:
warum? :)

Reply:
ach das ist der heroku branch. Ja das ist falsch => https://github.com/synyx/urlaubsverwaltung/blob/master/src/main/resources/static/favicon.ico